lib/networkConditions.js

```diff
@@ -0,0 +1,85 @@
+const NETWORK_PRESETS = [
```

Comment:
Can this be key value pair instead of array like 
```
'GPRS' : {
            offline: false,
            downloadThroughput: (50 * 1024) / 8,
            uploadThroughput: (20 * 1024) / 8,
            latency: 500
        }
```
this will avoid filtering on array.